# Laelaps laelaps_control/Pose2DStamped
#
# Robot 2D pose.
#
# A pose specifies position x,y and orientation theta. A timestamped header
# is also provided. A series of these poses define a path.
#

Header                header    # standard header
geometry_msgs/Pose2D  pose      # 2D pose
